meta/pattern-template.md

```diff
@@ -49,18 +64,23 @@ May mention:
 
 ## Status (optional until merging)
 
-General pattern status is stored in GitHub's Label tagging - see any pull request. Note that this GitHub label tagging becomes less visible once the pattern is finalized and merged, so having some information in this field is helpful.
+General pattern status is stored in GitHub's Label tagging - see any pull request.
+Note that this GitHub label tagging becomes less visible once the pattern is finalized and merged, so having some information in this field is helpful.
 
 You might store other related info here, such as review history: "Three of us reviewed this on 2/5/17 and it needs John's expertise before it can go further."
 
 ## Author(s) (optional)
 
-Often, this is yourself; If you need to, find someone in the InnerSource Commons to be the nominal author (As Told To); Could also be no-one if you do not want to take on authorship (common with a donut looking for a solution)
+Often, this is yourself;
+If you need to, find someone in the InnerSource Commons to be the nominal author (As Told To);
+Could also be no-one if you do not want to take on authorship (common with a donut looking for a solution)
```

Comment:
```suggestion
Often, this is yourself.
If you need to, find someone in the InnerSource Commons to be the nominal author (As Told To).
Could also be no-one if you do not want to take on authorship (common with a donut looking for a solution).
```

Changing this to regular sentences ending with a period (.) like everywhere else in this template.